README.md

```diff
@@ -312,32 +381,46 @@ Options and callback are optional. The options properties are:
 *PTZ.* Get supported coordinate systems including their range limitations for selected configuration. Extends corresponding
 configuration object
 
+### callback
+The callback function is in the form:
+```javascript
+(error, data, xml)
+```
+*xml* (the xml response) is optional.
+
 ## Supported methods
-* GetSystemDateAndTime
-* GetCapabilities
-* GetVideoSources
-* GetProfiles
-* GetServices
-* GetDeviceInformation
-* GetStreamUri
-* GetSnapshotUri
-* GetPresets
-* GotoPreset
-* RelativeMove
-* AbsoluteMove
-* ContinuousMove
-* Stop
-* GetStatus
-* SystemReboot
-* GetImagingSettings
-* SetImagingSettings
-* GetHostname
-* GetScopes
-* SetScopes
-* GetRecordings
-* GetReplayUri
+* getSystemDateAndTime
+* getCapabilities
+* getVideoSources
+* getProfiles
+* getServices
+* getDeviceInformation
+* getStreamUri
+* getSnapshotUri
+* getSnapshot
+* getPresets
+* gotoPreset
+* setPreset
+* removePreset
+* gotoHomePosition
+* setHomePosition
+* relativeMove
+* absoluteMove
+* continuousMove
+* stop
+* getStatus
+* systemReboot
+* getImagingSettings
+* setImagingSettings
+* getHostname
+* getScopes
+* setScopes
+* getRecordings
+* getReplayUri
```

Comment:
Thanks for updating this. Good catch. We should probably update all the in-line code documentation too but lets save that for another day ;-)